packages/opencensus-instrumentation-http/README.md

```diff
@@ -1,15 +1,10 @@
 # OpenCensus HTTP Instrumentation for Node.js
 [![Gitter chat][gitter-image]][gitter-url]
 
-OpenCensus HTTP Instrumentation allows the user to automatically collect trace data.
+OpenCensus HTTP Instrumentation allows the user to automatically collect trace data. This package is used by [OpenCensus Node.js](https://github.com/census-instrumentation/opencensus-node/tree/master/packages/opencensus-nodejs) to instrument HTTP.
```

Comment:
We should mention that this is for both servers and clients. (Same with HTTPS.)

Reply:
Great, updated that.